helm/chart/maesh/values.yaml

```diff
@@ -65,6 +65,10 @@ mesh:
   # Added so we can launch on nodes with restrictions
   nodeSelector: {}
   tolerations: []
+  # Additional deployment annotations (e.g. for jaeger-operator sidecar injection)
+  annotations: {}
```

Comment:
Just a question, the `jaeger-operator` sidecar injection is only required if our jaeger intergation/deployment is not used, right?

Reply:
If we follow the Helm chart [Guidelines](https://github.com/containous/maesh/blob/master/helm/chart/maesh/Guidelines.md) these options should be commented out. Does it make sense? wdyt?